site/source/tutorials/10XX_development_environment/index.md

```diff
@@ -0,0 +1,146 @@
+
+While git is not strictly necessary to use Dojo 2, a robust development environment also needs some sort of version control system. All of Dojo 2 uses git as the version control tool and we manage all of our code on GitHub. Several of READMEs and other instructions will assume that git is being used for version control. If it or another version control tool is not already installed, you can follow [these instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+{% task 'Initialize a git repository' %}
+
+{% instruction 'Run `git init` within the root `biz-e-corp` directory' %}
+
+This should log `Initialized empty Git repository in /path/to/biz-e-corp/.git/` in your console.
+
+{% instruction 'Create your first commit' %}
+
+Use `git add` to begin tracking files, and `git commit` to create a commit. For more information, refer to [these instructions](https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository).
+
+{% section %}
+
+## Configure your editor
+
+{% task 'Get an editor that supports Typescript.' %}
+
+Any code editor will allow you to work on a Dojo 2 project, but an editor that supports TypeScript will give you a richer development experience, as Dojo 2 is built with TypeScript and specifically designed to take advantage of its features. A list of editors with plugins that enable typescript support is [available here](https://github.com/Microsoft/TypeScript/wiki/TypeScript-Editor-Support).
+
+{% instruction 'Install Typescript support' %}
+
+In Atom, for example, this means installing [atom-typescript](https://atom.io/packages/atom-typescript). Visual Studio Code comes with typescript support by default, and for Vim try installing a combination of plugins for [language features](https://github.com/Quramy/tsuquyomi) and [syntax highlighting](https://github.com/leafgarland/typescript-vim).
+
+{% instruction 'Test error highlighting' %}
+
+Within the demo app, open `src/widgets/WorkerContainer.ts`. In it, there should be code to create an instance of the Worker widget on line 21. Add a fake property, e.g.:
+
+```typescript
+const workers = workerData.map((worker, i) => w(Worker, {
+  key: `worker-${i}`,
+  fakeProp: 'foo',
+  ...worker
+}));
+```
+
+Verify that your editor has highlighted `fakeProp` as an error:
+
+<p class="center">![error message in Atom](./resources/atom_ts_error.png)</p>
+
+{% instruction 'Go further: VS Code and tasks' %}
+
+You may skip to the next step if you are not interested in trying out VS Code.
+
+VS Code supports Typescript by default, is written is Typescript, and allows you to [configure tasks](https://code.visualstudio.com/docs/editor/tasks) which are then integrated into the IDE.
+
+To configure tasks:
+
+* Press <kbd>⌘⇧P</kbd>/<kbd>Ctrl⇧P</kbd> to bring up the command list.
+* Start typing <kbd>tasks</kbd> and select the command `Tasks: Configure Task Runner`
+
+<p class="center">![gif of opening tasks.json](./resources/task_setup.gif)</p>
```

Comment:
I think we want to avoid using markup for formatting

Reply:
I copied this from the other tutorials' use of images for consistency. Currently I think only the third and sixth tutorial also include images -- should I remove the centered `<p>` from those as well?

Reply:
@vansimke @itorrey thoughts on markup within markdown?